utils/models.py

```diff
@@ -14,9 +14,11 @@ class BaseAbstractModel(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
     is_deleted = models.BooleanField(default=False)
+    is_published = models.BooleanField(default=False)
```

Comment:
Hi @neelxie , I think this being here is affecting apps that shouldn't be affected by the change. With this change the models in the authentication and transaction apps now have the `is_published` property. Is this the  intended behavior?

Reply:
@Tevinthuku Thank you, I have changed this implementation.